hyperspy/axes.py

```diff
@@ -469,6 +508,30 @@ class AxesManager(t.HasTraits):
 
     def __init__(self, axes_list):
         super(AxesManager, self).__init__()
+        self.events = Events()
+        self.events.axes_changed = Event("""
```

Comment:
Why do we need an `axes_changed` event. Currently it doesn't seem to be used. Shouldn't any consumer of such an event connect to the relevant `DataAxis` event?